test/ContractTestContext.js

```diff
@@ -35,8 +39,11 @@ class ContractTestContext {
             this.vault.address,
             this.cdp.address,
             this.externalOUSD.address,
+            this.exchanger.address,
             this.treasurySigner.address,
         );
+        // Post init contracts
+        await this.exchanger.init(this.lvUSD.address, this.coordinator.address);
```

Comment:
If I understand correctly init also runs an `approve` function
Maybe we should verify that the correct contract got approved (I thing calling `allowance` allows you to do it)

Reply:
I'll add a test in ContractTestContext.test.